### PR TITLE
Update logo

### PR DESCRIPTION
**Pull request by Timothy-Edwards**
image

### CODE REVIEW

**Review by cebjyre — DISMISSED** [review]

**Comment by cebjyre** [comment]
probably should have favicon updated, but that can be a separate thing

**Comment by Timothy-Edwards** [comment]
> probably should have favicon updated, but that can be a separate thing

I can do that 👍🏼

**Comment by Timothy-Edwards** [comment]
QA - New logo and favicon applied

image

**Review by cebjyre — APPROVED** [review]

File: source/layouts/layout.erb [hunk 1]
```diff
@@ -31,7 +31,7 @@ under the License.
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
     <title><%= current_page.data.title || "API Documentation" %></title>
     <link href="https://fonts.googleapis.com/css?family=Roboto:400,500" rel="stylesheet">
-    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.png">
```
Comment (pfbaz) [inline source/layouts/layout.erb:34]: [B?] I think this means we need to add a `favicon.png` to the directory where `favicon.ico` is being removed from.

- [ ] Check and maybe add favicon.png unless it is magically fetched from somewhere else that I don't know about in which case :bushes: :rocket:

Comment (Timothy-Edwards) [reply source/layouts/layout.erb:34]: Great catch 🤦🏼 `git commit -am` vs `git add -A` got me. That should be fixed.

Comment (cebjyre) [reply source/layouts/layout.erb:34]: BOOO @Timothy-Edwards BOOOO

And this is why we `git add` separately from the `git commit`

**Review by cebjyre — APPROVED** [review]
this time for sure!

**Review by pfbaz — APPROVED** [review]

**Pull request merged**